src/tests/ftest/deployment/network_failure.py

```diff
@@ -180,15 +147,16 @@ def verify_network_failure(self, ior_namespace, container_namespace):
             "fabric_iface", "/run/server_config/servers/0/*")
         self.log.info("interface to update = %s", self.interface)
 
-        # wolf
-        self.update_network_interface(
-            interface=self.interface, state="down", host=self.network_down_host,
-            errors=errors)
-
-        # Aurora
-        # command = f"sudo ip link set {self.interface} down"
-        # self.log.debug("## Call %s on %s", command, self.network_down_host)
-        # time.sleep(20)
+        if self.test_env == "ci":
+            # wolf
+            update_network_interface(
+                interface=self.interface, state="down", host=self.network_down_host,
+                errors=errors)
+        else:
+            # Aurora
```

Comment:
ONLY if you have to push again, to be similar to other comments.
Question - Can this not be automated on Aurora?
```suggestion
            # Aurora. Manually run the command.
```

Reply:
In Aurora, sudo ip command works, but asks for the password, so I need to run this manually while other steps are automated.

Reply:
It might be worth requesting passwordless sudo access for the `ip` command. We can discuss in the next standup